README.md

```diff
@@ -1,2 +1,69 @@
-# windup-operator
-Windup Operator for Openshift-Kubernetes
+# windup-operator project
+
+This project creates an Operator to deploy Windup/MTA on an Openshift cluster ( Kubernetes one in future releases)
+
+It consists on a series of yaml files to deploy the operator (CRD, role, rolebinding, serviceaccount) and the native artifact compiled using Quarkus and GraalVM
+
+At this moment the operator reacts to creation of the Windup Custom Resource and will deploy all needed objects ( deployments, ingress, services, persistent volumes)
+
+## Building and pushing the Java code manually in native mode
+1. The following command will use the configuration in the `application.properties` file :  
+```
+quarkus.kubernetes.deployment-target=kubernetes, openshift
+quarkus.native.resources.includes=k8s/def/*.yaml
+quarkus.container-image.registry=quay.io
+quarkus.container-image.group=windupeng
+quarkus.container-image.name=windup-operator-native
+quarkus.container-image.tag=latest
+quarkus.kubernetes.service-type=load-balancer
+
+```
+1. Install JDK 11  
+You can use different ways , but SDKMan is very easy
+2. Execute the maven command:  
+`mvn clean package -Pnative -Dquarkus.native.container-build=true -Dquarkus.container-image.push=true`
```

Comment:
Some comments:

- `mvn` is the command used but also [`mvnw`](https://github.com/windup/windup-operator/pull/1/files#diff-71e4d281385254628e2f465d531483e29749cb5f970a14b177707285acf3478a) (and [`mvnw.cmd`](https://github.com/windup/windup-operator/pull/1/files#diff-64a82735d638ae1f0f66748c12366c49ab809d2a3efb6e99c451cace0f1c968c) as well) is available in this PR but it doesn't work due to the lack of `.mvn` subdirectory. With Quarkus is common to use `mvnw` but both `mvn` and `mvnw` seems fine. Hence:
  - or you delete `mvnw` and `mvnw.cmd` if we will go with `mvn`
  - or you change the command and add the `.mvn` subdirectory if we will adopt the other approach
- `quarkus.native.container-build` property requires [a container runtime](https://quarkus.io/guides/building-native-image#quarkus-native-pkg-native-config_quarkus.native.container-build). Docker is the default so it should be added as a dependency in the previous step together with JDK 11

Reply:
I agree with first one. I wil remove mvnw.cmd

Reply:
Second one is using JIB , as it is included in the POM, that doesnt need the Docker infrastructure installed.

Reply:
I will specify `podman` as the runtime as RH is going more into that direction.

Reply:
And change the README to add a step to install podman

Reply:
> Second one is using JIB to produce images but you are right that it needs Docker to run the GraalVM as it is pulling the image and running a `docker run`

`quarkus.native.container-build=true` requires a container runtime.
`quarkus.container-image.push=true` uses JIB.
My comment was only focused on `quarkus.native.container-build` property

Reply:
:+1